go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
Consider adding comments that specify the purpose of certain dependencies if they are not self-explanatory, which will help other developers understand the necessity of each package.